scifin/classifier/__init__.py

```diff
@@ -7,8 +7,11 @@
 The :mod:`scifin.classifier` module includes methods for classification and clustering.
 """
 
-from .classifier import (euclidean_distance, dtw_distance,
+from .classifier import (euclidean_distance, dtw_distance, dtw_distance_matrix_from_ts,
```

Comment:
Are you importing all functions & classes when initialising here? in that case, you could just use `from .classifier import *` here

Reply:
I agree with your comment. But actually I come to realize that the function make_new_outputs() is probably not going to be used on its own (it is only used in kmeans_advanced_clustering()), so I am going to remove it from the list.